test/functional/cmdLineTests/cmdLineTest_J9tests/build.xml

```diff
@@ -30,16 +30,43 @@
 
 	<!-- set properties for this build -->
 	<property name="DEST" value="${BUILD_ROOT}/functional/cmdLineTests/cmdLineTest_J9tests" />
-	<property name="src" location="." />
+	<property name="PROJECT_ROOT" location="." />
+	<property name="src" location="${PROJECT_ROOT}/src" />
+	<property name="build" location="${PROJECT_ROOT}/bin" />
 
-	<target name="dist" description="generate the distribution">
+	<target name="init">
+		<mkdir dir="${DEST}" />
+		<mkdir dir="${build}" />
+	</target>
+
+	<target name="compile" depends="init" description="Using java ${JDK_VERSION} to compile the source ">
+		<echo>Ant version is ${ant.version}</echo>
+		<echo>============COMPILER SETTINGS============</echo>
+		<echo>===fork:                         yes</echo>
+		<echo>===executable:                   ${compiler.javac}</echo>
+		<echo>===debug:                        on</echo>
+		<echo>===destdir:                      ${DEST}</echo>
```

Comment:
@keithc-ca Tabs for the echo text have been switched to spaces. This whitespace change was added to each `build.xml` file in the `test/functional` directory with similar "COMPILER SETTINGS" echo messages.

Reply:
Whoops, need to fix the copyrights for each of the updated files.

Reply:
The spacing changes to files that were otherwise not modified for this PR should be made in a separate PR.